### DISCUSSION TITLE
Ignore Type

### DISCUSSION THREAD

**Discussion by evancjohnson** [IEvangelist/azure-cosmos-dotnet-repository#281] [Q&A]
Hello Friends,

I've just started using this repository to take place of one that i've already written. My only problem so far is this library uses a custom type per entity which is perfectly fine. The only issue is I already have a type that i've been using. This has caused a couple of issues with some queries for things that have a "Type" already defined.

Should I just re-write these with plain queries?
Does a method exist to disable auto adding type to `GetAsync()` calls?

**Comment by mumby0168** [comment]
Hi Evan,

What is your Type field currently used for?
